docs/pelion_ready/device-management.md

```diff
@@ -17,30 +17,30 @@ This guide builds on the [quick connect guide](https://os.mbed.com/guides/connec
 - [Validation and testing](../mbed-os-pelion/device-management-test.html).
 - [Troubleshooting](../mbed-os-pelion/device-management-test.html#troubleshooting).
 
-<span class="tips">**Tip**: To learn more about Device Management Client, please see our [Device Management documentation](https://cloud.mbed.com/docs/latest/client-api-references/index.html).</span>
+<span class="tips">**Tip**: To learn more about Device Management Client, see the [API documentation](https://www.pelion.com/docs/device-management/current/client-api-references/index.html) and the [tutorials](https://www.pelion.com/docs/device-management/current/connecting/device-management-client-tutorials.html).</span>
 
 ## Device Management for your Mbed OS application
 
 ### Requirements
 
-Not every device (microcontroller, module or board) is capable of running device management features. Although you can add or extend some hardware capabilities, such as connectivity, storage and TRNG, others are impossible or inconvenient to extend (for example, RAM or flash).
+Not every device (microcontroller, module or board) is capable of running device management features. Although you can add or extend some hardware capabilities, such as connectivity, storage and TRNG. Others are impossible or inconvenient to extend (for example, RAM or flash).
```

Comment:
Device management or Device Management? Are we talking general concepts or our product in particular?